GOVERNANCE.md

```diff
@@ -1,10 +1,10 @@
 # OpenJS Cross Project Council
 
-For the current list of Team members, see the project [README.md][README] and the [OpenJS Foundation Directory](https://github.com/openjs-foundation/directory-private/blob/master/cpc-private.md) (restricted access for privacy reasons).
+For the current list of Team members, see the project [README.md][README] and the [OpenJS Foundation Directory][] (restricted access for privacy reasons).
```

Comment:
@brianwarner the governance references the OpenJS Foundation Directory twice but they are two different files. Can you clarify that both are correct? One is the markdown file and one is the yaml file (further below at line 149). If you could just double check me on these, that'd be great. Thanks!